setup.py

```diff
@@ -21,7 +21,7 @@
     description="Python library for interacting with the Faculty platform.",
     url="https://sherlockml.com",
     author="Faculty",
-     author_email="opensource@faculty.ai",
+    author_email="opensource@faculty.ai",
     license="Apache Software License",
```

Comment:
GitHub suggestions must be buggy, it didn't show any change in indentation:
![screenshot 2019-02-05 at 16 03 20](https://user-images.githubusercontent.com/5845679/52286233-c1b4e300-295f-11e9-8115-89292b63fbdf.png)